### ISSUE TITLE
File not allowed: FSO2014H.csv.hdr

### ISSUE THREAD

**Issue by ClaytonJY** [javadch/rquis#7]
From `R CMD check`:

```
* checking contents of ‘data’ directory ... WARNING
Files not of a type allowed in a ‘data’ directory:
  ‘FSO2014H.csv.hdr’
Please use e.g. ‘inst/extdata’ for non-R data files
```

I think the idea here is to show how one can use a separate file to specify headers, but it needs to be organized different in the R package. The suggestion to use `inst/extdata` may or may not be the best option. See [Hadley's guide](http://r-pkgs.had.co.nz/data.html) for more.

**Comment by javadch** [comment]
If the inst/extdata is accessible through the XQt example scripts, lets move the data there

**Comment by ClaytonJY** [comment]
I agree, as commented in #4 .

**Comment by ClaytonJY** [comment]
Fixed with #13.
